### DISCUSSION TITLE
Wave pattern in UBL

### DISCUSSION THREAD

**Discussion by bjoernbusch** [Jyers/Marlin#590] [General Discussion]
I see this when I look at the bed level using UBL 10x10:

<img width="734" alt="Screenshot 2021-04-25 at 07 16 40" src="https://user-images.githubusercontent.com/15795611/115981750-64d09200-a596-11eb-94a2-1f5fe86fd818.png">

I have been trying to figure this out for quite a while now, but I can't pinpoint it. I've exchanged the rolls, taken apart the printer 3 times and made sure everything runs smooth and is angled properly. I've tested the BLtouch with the accuracy command and it looks fine. I'm using the standard Ender 3 v2 glasbed, which I know has a dip in the middle, but not a wave pattern. When I rotate it 90 degrees, I get a similar pattern, when I use another FR4 bed, it's also similar, so it must be something else.

Any hints from anybody?

**Comment by darco415** [comment]
I wouldnt put too much emphasis on getting  a perfectly flat surface when you have a BLT. I mean thats what its for.
